tutorials/10-sprite-sheet/src/font_sprite.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+# _____        ____   ___
+#   |     \/   ____| |___|
+#   |     |   |   \  |   |
+#-----------------------------------------------------------------------
+# Copyright 2022-2023, tyra - https://github.com/h4570/tyra
+# Licensed under Apache License 2.0
+# Guido Diego Quispe Robles
+*/
+
+#include "font_sprite.hpp"
+
+namespace Tyra {
+
+const int Font::chars[FONT_CHAR_SIZE]{

```

Comment:
I changed some letters, because of:

![image](https://github.com/h4570/tyra/assets/32263891/cc3d2dc0-1424-4796-980d-ebf1791bc9b7)

Reply:
It's fine, no problem